src/shared_modules/utils/socketWrapper.hpp

```diff
@@ -28,7 +28,7 @@
 #include <thread>
 #include <unistd.h>
 
-constexpr auto INVALID_SOCKET {-1};
+constexpr auto _INVALID_SOCKET {-1};
```

Comment:
Question: Is there a reason for the variable name change?

Reply:
There's a collision with a preprocessor directive in httplib.

Reply:
Fair enough, I've didn't found any good solution for this

Maybe using #undef but it could lead to problems